code/game/machinery/autolathe.dm

```diff
@@ -374,7 +374,7 @@
 	for(var/datum/design/D in files.possible_designs)
 		if((D.build_type & AUTOLATHE) && ("hacked" in D.category))
 			if(hacked)
-				files.known_designs += D
+				files.known_designs |= D

```

Comment:
At worst you awakened existing shitcode.  It should always have been ```|=```.  git blame shows this particular line dated to 2014 though:

```
04fda279 (Menshin                      2014-12-13 22:43:41 +0100 377)                           files.known_designs += D
```